pkg/defaultRules/defaultRules.yaml

```diff
@@ -1869,3 +1869,96 @@ rules:
         $ref: "#"
       items:
         $ref: "#"
+  - id: 61
+    name: "Prevent setting 'Unconfined' as seccomp profile"
+    uniqueName: CONTAINERS_INCORRECT_SECCOMP_PROFILE
+    enabledByDefault: false
+    documentationUrl: "https://www.datree.io/resources/eks-best-practices-you-should-know"
+    messageOnFailure: "seccompProfile was omitted or was explicitly set to 'Unconfined' - set an explicit value to increase the pod or container's isolation"
```

Comment:
Also, to keep a consistent format of our rule docs, I think this should be:
"Incorrect value for key `seccompProfile` - set an explicit value to prevent malicious use of system calls within the container"